docs/migrations/25-04.md

```diff
@@ -37,6 +37,35 @@ The third preview of workflow outputs introduces the following breaking changes
 
 See {ref}`workflow-output-def` to learn more about the workflow output definition.
 
+<h3>Workflow params (first preview)</h3>
+
+The `params` block is a new way to declare pipeline parameters in a Nextflow script:
```

Comment:
there's already support for parameters. It should be highlighted better what's new with this preview

Reply:
The benefit is further down:

> This syntax allows you to declare all parameters in one place and allows Nextflow to validate them at runtime.